sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -405,7 +405,7 @@ def _set_access_control_recursive_options(mode, acl, **kwargs):
             'mode': mode,
             'force_flag': kwargs.pop('continue_on_failure', None),
             'timeout': kwargs.pop('timeout', None),
-            'continuation': kwargs.pop('continuation', None),
+            'continuation': kwargs.pop('continuation', None) or kwargs.pop('continuation_token', None),
```

Comment:
why do we pop both continuation and continuation_token ?

Reply:
Because we'd better not break the existing customers code, so I'd rather keep what we were already support in preview.
Let me know if we want to delete one!